solana-programs/anchor/audius-data/Anchor.toml

```diff
@@ -6,7 +6,7 @@ url = "https://anchor.projectserum.com"
 
 [provider]
 cluster = "localnet"
-wallet = "/Users/hareeshnagaraj/.config/solana/id.json"
+wallet = "/Users/cheran/.config/solana/id.json"
```

Comment:
@hareeshnagaraj we should probably make this part better

Reply:
yeah this shouldn't have to be a direct path - mind adding a note?

Reply:
Done